Common/include/CConfig.hpp

```diff
@@ -2279,8 +2280,7 @@ class CConfig {
     string FEAFilename = FEA_FileName;
 
     /*--- strip the extension if it is present, only if it is .dat ---*/
-    auto extIndex = FEAFilename.rfind(".dat");
-    if (extIndex != std::string::npos) FEAFilename.resize(extIndex);
+    PrintingToolbox::TrimExtension(".dat", FEAFilename);
```

Comment:
Use this helper please, it checks that the extension is actually at the end of the file